db/__tests__/seed.test.js

```diff
@@ -48,7 +48,7 @@ test('GET 200: responds with an object of endpoints"', () => {
 })
 
 describe('/api/articles/:article_id', () => {
-test('GET 200: responds with the article that matches the given article_id', () => {
+test('GET 200: responds with the article that matches the given article_id and comment_count of the number of comments with the given article_id', () => {
```

Comment:
We shouldn't be going back to edit old tests when we add new functionality, the ideal way is to create a new test asserting against just the new property that we have now added by default to this endpoint. Both tests should pass if we are correctly using flexible jest matchers.